### PR TITLE
[12.0] [ADD] Account Payment Return Import ISO20022

### PR DESCRIPTION
**Pull request by ThomasBinsfeld**
New addon `account_payment_return_import_iso20022` replacing the existing `account_payment_return_import_sepa_pain` and adding the possibility to import camt.054.001.02 files.

------------------
From #235

### CODE REVIEW

File: account_payment_return_import_iso20022/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    'license': 'AGPL-3',
+    'author': 'Odoo Community Association (OCA),Tecnativa,ACSONE SA/NV',
+    'website': 'https://github.com/OCA/account-payment/tree/12.0/'
+               'account_payment_return_import_iso20022',
+    'depends': [
+        # OCA/account-payment
+        'account_payment_return',
+        'account_payment_return_import',
+        # OCA/bank-payment
+        'account_payment_order',
```
Comment (pedrobaeza) [inline account_payment_return_import_iso20022/__manifest__.py:20]: Why do you need this new dependency?

Comment (ThomasBinsfeld) [reply account_payment_return_import_iso20022/__manifest__.py:20]: It's not new. It was already in `account_payment_return_import_sepa_pain`.

Comment (pedrobaeza) [reply account_payment_return_import_iso20022/__manifest__.py:20]: My bad, sorry

**Review by pedrobaeza — APPROVED** [review]

**Review by chrisdec — APPROVED** [review]
Functional tests

**Comment by pedrobaeza** [comment]
Renaming for OpenUpgrade path in https://github.com/OCA/OpenUpgrade/commit/4a4da2c62d3f2deb34c733ca1e82b6b39227aff6

**Pull request merged**